components/docs-chef-io/content/automate/node_bootstarp_ha.md

```diff
@@ -0,0 +1,148 @@
+
+To bootstrap a node, run `knife bootstrap` in the workstation.
+
+## Knife Bootstrap
+
+The knife bootstrap command is a common way to install Chef Infra Client on a node. The default for this approach assumes that a node can access the Chef website so that it may download the Chef Infra Client package from that location.
+
+The Chef Infra Client installer will detect the version of the operating system and then install the appropriate Chef Infra Client version using a single command to install Chef Infra Client and all of its dependencies, including an embedded version of Ruby, OpenSSL, parsers, libraries, and command-line utilities.
+
+The Chef Infra Client installer puts everything into a unique directory (/opt/chef/) so that Chef Infra Client will not interfere with other applications that may be running on the target machine. Once installed, Chef Infra Client requires a few more configuration steps before performing its first Chef Infra Client run on a node.
+
+## Bootstrap a Node
+
+### Create Users and Organization
+
+1. ssh in the bastion host and follow the below steps.
+
+2. ssh in chef_server instance.
+
+`sudo chef-automate ssh --hostname chef_server`
+
+3. Ensure all the front-end instances are up and running.
+
+`sudo chef-automate status`
+
+4. Create a User.
+
+Syntax:
+
+```bash
+sudo chef-server-ctl user-create <username> <First Name> <Last Name> <Email ID> <password> -f <path and file name to store user's pem file>
+```
+
+For Example:
+
+```bash
+sudo chef-server-ctl user-create johndoe John Doe john.doe@example.com John@123 -f ./johndoe.pem
+```
+
+Created users can be listed using `sudo chef-server-ctl user-list`
+
+5. Create an organization.
+
+Syntax:
+
+```bash
+sudo chef-server-ctl org-create <org name> '<org display name>' --association_user <username> -f <path and file name to store org's pem file>
+```
+
+For Example:
+
+```bash
+sudo chef-server-ctl org-create new_org 'New Organization' --association_user johndoe -f ./new_org.pem
+```
+
+Created organization can be listed using `sudo chef-server-ctl org-list`
+
+7. Copy the `pem` files and save them to a safe location.
+
+### Workstation Setup
+
+1. To set up the workstation on your machine, follow the steps given below:
+
+-   Install the latest version of chef Workstation on the ubuntu system.
+
+```bash
+wget https://packages.chef.io/files/stable/chef-workstation/21.7.524/ubuntu/20.04/chef-workstation_21.7.524-1_amd64.deb
+```
+
+-   To install the same:
+
+```bash
+dpkg -i chef-workstation_21.7.524-1_amd64.deb
+```
+
+-   Verify the installation using following command:
+
+```bash
+chef -v
+```
+
+Click [here](https://docs.chef.io/workstation/install_workstation/) for any additional information.
+
+2. Generate chef-repo using `chef generate repo chef-repo`. Click [here]https://docs.chef.io/workstation/getting_started/ to know more.
+
+3. Paste `pem` file of user inside `/root/.chef/<pem_file_of_user>`. For example: `Eg.: /root/.chef/johndoe.pem`
```

Comment:
we should also ask to put org pem file in /root/.chef/<pem_file_of_org>

Reply:
Added